src/sentry/integrations/custom_scm/integration.py

```diff
@@ -0,0 +1,141 @@
+from uuid import uuid4
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.constants import ObjectStatus
+from sentry.integrations import (
+    FeatureDescription,
+    IntegrationFeatures,
+    IntegrationInstallation,
+    IntegrationMetadata,
+    IntegrationProvider,
+)
+from sentry.integrations.repositories import RepositoryMixin
+from sentry.models.repository import Repository
+from sentry.pipeline import PipelineView
+from sentry.web.helpers import render_to_response
+
+from .repository import CustomSCMRepositoryProvider
+
+DESCRIPTION = """
+Custom Source Control Management (SCM)
+"""
+
+FEATURES = [
+    FeatureDescription(
+        """
+        Send your own commits
+        """,
+        IntegrationFeatures.COMMITS,
+    ),
+    FeatureDescription(
+        """
+        Stack trace linky dink
```

Comment:
@MeredithAnya we probs should have some real copy here

Reply:
yeah I'm going just wait until the end for this, I don't think we need to do this now